customisation/glossaries.md

```diff
@@ -5,3 +5,45 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
```

Comment:
Actually Microsoft offers a functionality called "Dictionary-only training" where you launch a "training" with a glossary only. https://docs.microsoft.com/en-us/azure/cognitive-services/translator/custom-translator/what-is-dictionary#dictionary-only-trainings
I guess it's some sort of fine-tuning of their generic model with a glossary, but since they call it "training" I was wondering whether it makes sense that "training" appears in the title? Perhaps that's why it was there in the first place?

Reply:
Interesting, good to know

"customizing" is more general so it is true for all the engines.

It lets us avoid asserting whether model training happens or not.